app/resources/api/v2/shared_behaviour/receptacle.rb

```diff
@@ -35,6 +35,8 @@ module Api::V2::SharedBehaviour::Receptacle
     has_many :transfer_requests_as_source, readonly: true
     has_many :transfer_requests_as_target, readonly: true
 
+    has_one :labware, readonly: true
```

Comment:
curious why that is required. 
all receptacles have always linked to one labware and it hasn't been needed before

Reply:
I am using this to access labware and subsequently labware_barcode to get the barcode information. I think I could add a labware_barcode definition in receptacle, same as in labware but not sure thats needed.